databricks/koalas/frame.py

```diff
@@ -3225,6 +3225,58 @@ def mask(self, cond, other=np.nan) -> "DataFrame":
         cond_inversed = cond._apply_series_op(lambda kser: ~kser)
         return self.where(cond_inversed, other)
 
+    # TODO: Support axis as 1 or 'columns'
+    def mode(
+        self, axis: Union[int, str] = 0, numeric_only: bool = False, dropna: bool = True
+    ) -> "DataFrame":
+        """
+        Get the mode(s) of each element along the selected axis.
+
+        The mode of a set of values is the value that appears most often.
+        It can be multiple values.
+
+        Parameters
+        ----------
+        axis : {0 or 'index', 1 or 'columns'}, default 0
+            The axis to iterate over while searching for the mode:
+
+            * 0 or 'index' : get mode of each column
+            * 1 or 'columns' : get mode of each row.
+
+        numeric_only : bool, default False
+            If True, only apply to numeric columns.
+        dropna : bool, default True
+            Don't consider counts of NaN/NaT.
+
+        Returns
+        -------
+        DataFrame
+            The modes of each column or row.
+
+        See Also
+        --------
+        Series.mode : Return the highest frequency value in a Series.
+        Series.value_counts : Return the counts of values in a Series.
+        """
+        axis = validate_axis(axis)
+        assert axis == 0
+
+        def mode(kser, dropna):
```

Comment:
It fails in CI jobs because of `TypeError: ("mode() got an unexpected keyword argument 'dropna'", 'occurred at index a')`. However, it runs well locally. May I ask if you know the reason by any chance? @ueshin